smart_contracts/contract/src/ext_ffi.rs

```diff
@@ -700,6 +700,11 @@ extern "C" {
         out_ptr: *mut u8,
         out_size: usize,
     ) -> i32;
+    ///
```

Comment:
This one stands out without rustdoc

Reply:
I'm following the example of the other `load_` function. :stuck_out_tongue_winking_eye: